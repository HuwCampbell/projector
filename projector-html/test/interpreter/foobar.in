<a>{ text "foo! bar! baz!"}{ blank }{ text "QUUX" }</a><b/>