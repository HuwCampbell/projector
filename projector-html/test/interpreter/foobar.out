Nested
  [ Element "a" [] (Nested [ Plain "foo! bar! baz!" , Plain "QUUX" ])
  , VoidElement "b" []
  ]